src/main/java/Database/DatabaseController.java

```diff
@@ -41,17 +42,21 @@ public static String tableToString(Table tb) {
     }
   }
 
+  public static void main(String[] args) {
```

Comment:
do we need this main method here? if it's just for testing make it a test case or remove it